datahub-web-react/src/app/ingest/source/builder/RecipeForm/FormField.tsx

```diff
@@ -80,6 +84,30 @@ function ListField({ field, removeMargin }: ListFieldProps) {
     );
 }
 
+function SelectField({ field }: SelectFieldProps) {
+    return (
+        <Form.Item
+            name={field.name}
+            label={
+                <Label>
+                    {field.label}
+                    <Tooltip overlay={field.tooltip}>
+                        <StyledQuestion />
+                    </Tooltip>
+                </Label>
+            }
```

Comment:
here you can actually just let ant design do this work for you! so you can just pass in `label={field.label}` and `tooltip={field.tooltip}`

Reply:
Checked and fixed.